go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/azure-workload-identity
 
-go 1.18
+go 1.19
```

Comment:
why are we bumping go directives? do we require 1.19 to build?

Reply:
we're bumping the base image used to build to 1.19, so this change would be consistent with that!

Reply:
Docs are pretty ambiguous about this but it's meant as a min version so if we were using a specific feature in 1.19 then we would update this, otherwise keep the same version.